rest-api/src/main/java/gov/cms/qpp/conversion/api/services/CpcFileService.java

```diff
@@ -0,0 +1,8 @@
+package gov.cms.qpp.conversion.api.services;
+
+import gov.cms.qpp.conversion.api.model.UnprocessedCpcFileData;
+import java.util.List;
+
+public interface CpcFileService {
+	public List<UnprocessedCpcFileData> getUnprocessedCpcPlusFiles();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "public" is redundant in this context. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2333)